easybuild/easyconfigs/b/BWA/BWA-0.7.17.20220923-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,59 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Cyprus Institute / CaSToRC, Uni.Lu/LCSB, NTUA
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+#
+# Version >= 0.7.15
+# Author: Adam Huffman
+# The Francis Crick Institute
+#
+# Note that upstream development is mainly at: https://github.com/lh3/bwa
+##
+
+name = 'BWA'
+version = '0.7.17.20220923'
+
+homepage = 'http://bio-bwa.sourceforge.net/'
+
+description = """
+ Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively
+ short nucleotide sequences against a long reference sequence such as the human
+ genome.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+local_commit = '139f68f'
+
+sources = [{
+    'filename': '%s-%s-%s.tar.gz' % (name, version, local_commit),
+    'git_config': {
+        'url': 'https://github.com/lh3',
+        'repo_name': 'bwa',
+        'commit': local_commit,
+        'recursive': True,
+    },
+}]
+checksums = ['127d3f82ccc3bf7e918d2e64cafc0ffc49ff841b0955327f4666ca448364859f']
```

Comment:
The significant downside of using `git_config` is that we can't use a checksum, because the source tarball is created locally on the fly and it won't be exactly the same across systems, see https://github.com/easybuilders/easybuild-framework/issues/2726

In this case, I don't think we need to use `git_config` at all, we can just download the commit directly from GitHub as a tarball.

Fixed in https://github.com/hvelab/easybuild-easyconfigs/pull/1